src/lib/operators/table_scan/single_column_table_scan_impl.cpp

```diff
@@ -61,13 +78,79 @@ void SingleColumnTableScanImpl::handle_value_column(const BaseValueColumn& base_
   });
 }
 
-void SingleColumnTableScanImpl::handle_dictionary_column(const BaseDictionaryColumn& base_column,
+void SingleColumnTableScanImpl::handle_dictionary_column(const BaseDeprecatedDictionaryColumn& base_column,
                                                          std::shared_ptr<ColumnVisitableContext> base_context) {
+  _handle_dictionary_column(base_column, base_context);
+
+  // Implementation of an unoptimized scan
```

Comment:
Any way we can have this as actual, compiled code? Otherwise, people will change the code that is compiled and this comment will quickly become outdated.

Reply:
I removed it.